packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,26 @@
+# Best Practices
+
+A flat select allows users to select a single value among multiple options.
+
+The width of each of the options should be the same, we don't want any of the options feel more important than the other.
+
+Ideally, offer a maximum of five options.
+
+Don't use Flat select when the effect is not immediate, consider using dropdown menu, radio buttons or checkboxes instead. Users are expect to have immediate feedback on their actions when using Flat select.
+
+If used as view switcher (tabs), make sure it makes sense hierarchically and that tabs cannot be used.
+
+## Labeling
+- Keep titles short, preferably **under three words**.
+- Write for scannability
+
+## Flat select vs Toggle switch vs Tabs
+The Flat select share similarities with various other components but here we try to clarify the difference between some of them.
+
+## Flat select vs Tabs
+Both are similar in appearance and designed for view switching on an element. Flat select are suitable for view switching within a pane, as where tabs, are used as a navigational control (Moving from screen to another).
+
+## Related Components
+- [Tabs](https://plasma.coveo.com/navigation/Tabs) Might be used instead to toggle views
+- [Radio buttons](https://plasma.coveo.com/form/RadioButton) Can be used instead as a form input
+- [Select - single](https://plasma.coveo.com/form/SingleSelect) Can be used instead as a form input
```

Comment:
```suggestion
## Related Components

- [Radio buttons](https://plasma.coveo.com/form/RadioButton) - Alternative to a flat select in the context of a form
- [Single select](https://plasma.coveo.com/form/SingleSelect) - Alternative to a flat select in the context of a form
- [Tabs](https://plasma.coveo.com/navigation/Tabs) - Alternative to a toggle switch in a view switching context
```
@johnnyCoveo I feel we should give more details on what should be used when. They're probably not perfect equivalents? For instance, maybe there's a difference in the number of options, space required...?

Reply:
@jlevesquecoveo Makes sense, except line 27, we should read "Alternative to a flat select.."

Reply:
Thanks, good catch. Should we also add the checkbox, drop-down menu and toggle switch to this list, since we mention them in the article?

Also, the article didn't clarify what the difference between a single select and a flat select is. I think it should be stated here, in place of "Alternative to a flat select in the context of a form." Let me know!